.isort.cfg

```diff
@@ -1,5 +0,0 @@
-[settings]
-known_first_party=argoverse
-# cat setup.py | grep -Pzo "(?s)install_requires.+?]" | tail -n+2 | head -n-1 | grep -Po '"[a-zA-Z0-9]+' | sed 's/"/,/'
-known_third_party=mayavi,pytest,colour,descartes,imageio,matplotlib,motmetrics,numpy,cv2,pandas,pillow,imageio,pyntcloud,scipy,shapely,sklearn
```

Comment:
I believe this doesn't affect the installation of these packages, right?

Reply:
That's right. I added them back in for now.